deploy/helm/kubecf/values.yaml

```diff
@@ -249,12 +143,7 @@ settings:
 
 features:
   eirini:
-    # When eirini is enabled, both suse_default_stack and suse_buildpacks must be enabled as well.
     enabled: false
-    registry:
-      service:
-        # This setting is not currently configurable and must be HIDDEN
-        nodePort: 31666
```

Comment:
Not sure what replaces this right now - may have been fixed in the eirini helm chart?

Reply:
The setting was moved to `config/eirini.yaml`. This wouldn't have worked because the Eirini subchart doesn't use the new config system yet. However, the Eirini subchart never had access to this setting anyways, so it was completely unused.

I've removed the setting from `config/eirini.yaml` in e31167b.